backend/capellacollab/settings/modelsources/t4c/license_server/routes.py

```diff
@@ -25,11 +24,23 @@
 router = fastapi.APIRouter()
 
 
-@admin_router.get("", response_model=list[models.T4CLicenseServer])
+@router.get("")
 def get_t4c_license_servers(
     db: orm.Session = fastapi.Depends(database.get_db),
-) -> abc.Sequence[models.DatabaseT4CLicenseServer]:
-    return crud.get_t4c_license_servers(db)
+    user: users_models.DatabaseUser = fastapi.Depends(
+        users_injectables.get_own_user
+    ),
+) -> list[models.T4CLicenseServer]:
+    license_servers = [
+        models.T4CLicenseServer.model_validate(license_server)
+        for license_server in crud.get_t4c_license_servers(db)
+    ]
+
+    if user.role != users_models.Role.ADMIN:
```

Comment:
The lines are not covered by tests. Can you add another test that sends a request to the route as normal user (not admin) and check that it's also anonymized properly?